lib/rules/no-script-url.js

```diff
@@ -31,18 +33,26 @@ module.exports = {
 
     create(context) {
 
+        /**
+         * Check whether a node's static value starts with "javascript:" or not.
+         * And report an error for unexpected script URL.
+         * @param {ASTNode} node node to check
+         * @returns {void}
+         */
+        function check(node) {
+            const value = astUtils.getStaticStringValue(node);
+
+            if (typeof value === "string" && value.toLocaleLowerCase().indexOf("javascript:") === 0) {
```

Comment:
Could the change from `toLowerCase` to `toLocaleLowerCase` maybe produce more errors (which might be false positives)?

For instance, by the examples on [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleLowerCase) it looks like `"javascrİpt".toLocaleLowerCase()` could become `"javascript"` in some environments, e.g. with "tr" locale.

Reply:
@mdjermanovic  Thanks :) It was my mistakes 😅 I fixed it